Ginger/GingerCoreNET/DataSource/LiteDB.cs

```diff
@@ -720,21 +720,28 @@ public override void RenameTable(string tableName, string newTableName)
         {
             bool renameSuccess = false;
             bool tableExist = false;
-            using (var db = new LiteDatabase(FileFullPath))
+            try
```

Comment:
Check if new table name is empty in if condition instead of handling it in Try catch